tests/test_game.py

```diff
@@ -16,13 +16,13 @@ def test_game_initialization(self):
 
     def test_is_valid(self):
 
-            test_grid = list('ETIAGDNNERNIEGABLOBL')
+            test_grid = list('DSREOOVEJSUSOLF')
             # setup
             game = Game()
             # exercise
             game.grid = test_grid
             # verifiy
-            test_word = 'DATAENGINEERING'
+            test_word = 'OVERDOSE'
             assert game.is_valid(test_word) is True
 
     def test_is_not_valid(self):
```

Comment:
```suggestion
   def test_letter_countself):

            test_grid = list('GROVEABC')
            # setup
            game = Game()
            # exercise
            game.grid = test_grid
            # verifiy
            test_word = 'GROOVE'
            assert game.is_valid(test_word) is False

    def test_is_not_valid(self):
```

Reply:
thx addedd additional test for case like this